.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ion-object-mapper-dotnet"]
+	path = ion-object-mapper-dotnet
+	url = git@github.com:amzn/ion-object-mapper-dotnet.git
```

Comment:
Think we should hold off on doing this until we get onto nuget.

Reply:
Yup, this PR won't be merged until the object mapper is on NuGet and we remove the submodule.